scripts/release/package.json

```diff
@@ -1,12 +1,13 @@
 {
   "private": true,
   "dependencies": {
-    "chalk": "2.4.1",
-    "execa": "0.10.0",
-    "minimist": "1.2.3",
+    "chalk": "4.1.0",
+    "execa": "4.0.3",
+    "minimist": "1.2.5",
     "node-fetch": "2.6.1",
     "outdent": "0.7.1",
-    "semver": "5.5.0",
-    "string-width": "2.1.1"
-  }
+    "semver": "7.3.2",
+    "string-width": "4.2.0"
+  },
+  "version": "3.0.0"
```

Comment:
question, why did you add `version` ?

Reply:
I forgot... Removed https://github.com/prettier/prettier/pull/9418/commits/7ed0cca9cef4b42685a95321b63360403fe77148.